### ISSUE TITLE
Test suite failure

### ISSUE THREAD

**Issue by snoyberg** [brendanhay/ede#10]
```
Linking dist/build/golden/golden ...
Running 1 test suites...
Test suite golden: RUNNING...
golden: test/resources/: getDirectoryContents: does not exist (No such file or directory)
Test suite golden: FAIL
Test suite logged to: dist/test/ede-0.2.5-golden.log
0 of 1 test suites (0 of 1 test cases) passed.
```

**Comment by brendanhay** [comment]
Fixed by d60345e.

**Comment by brendanhay** [comment]
Reopening due to failure to include the `test/resources` other-files update in the release.
